Tasks/difference.js

```diff
@@ -1,19 +1,11 @@
 // Find a difference between two dictionaries
+'use strict';
 
-const diff = (DIFF = function diff(object_1, object_2) {
-  for (attribute_name in object_1) {
-    object_1[attribute_name] = object_1[attribute_name];
+const dictDifference = ({...obj1}, obj2) => {
+  for (const key in obj2) {
+    delete obj1[key];
```

Comment:
It will be better to add keys for new result object than `delete` kays, I'd prefer to avoid `delete` in most cases if we have a choice